CHANGES.md

```diff
@@ -20,12 +20,9 @@ Release Notes.
 * Fix MAL concurrent execution issues
 
 #### UI
-* Fix un-removed tags in trace query.
-* Fix unexpected metrics name on single value component.
-* Don't allow negative value as the refresh period.
-* Fix style issue in trace table view.
-* Separation Log and Dashboard selector data to avoid conflicts.
-* Fix trace instance selector bug.
```

Comment:
I think you should keep these? I did another UI sync, which includes all these changes.

Reply:
Okay